Sources/Core/ISO8601.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+// Modified from http://stackoverflow.com/a/28016692/239380
+
+public struct ISO8601 {
+    public static let shared = ISO8601()
+    public let formatter: DateFormatter
+
+    public init() {
+        formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .iso8601)
+        formatter.locale = Locale(identifier: "en_US_POSIX")
```

Comment:
Looks like I should add this: ` [rfc3339DateFormatter setTimeZone:[NSTimeZone timeZoneForSecondsFromGMT:0]];`, *converted to swift of course*.